### PR TITLE
Add translations to cookies, privacy policy and feedback view models

### PR DESCRIPTION
**Pull request by dbgoaco**
[DP-999](https://noticingsystem.atlassian.net/jira/software/projects/DP/boards/22?selectedIssue=DP-999)

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 @page "/provide-feedback-and-contact"
 @model ProvideFeedbackAndContact
+@using Localization;
+
 @{
     if (Model.Context == ContactFormTypes.Feedback)
     {
-        ViewData["Title"] = "Provide Find a Tender Feedback";
+        ViewData["Title"] = @StaticTextResource.ProvideFeedbackAndContact_ProvideFindATenderFeedback;
```
Comment (rmohammed-goaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:8]:
```suggestion
        ViewData["Title"] = StaticTextResource.ProvideFeedbackAndContact_ProvideFindATenderFeedback;
```

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
     }
     else if (Model.Context == ContactFormTypes.Support)
     {
-        ViewData["Title"] = "Contact Find a Tender";
+        ViewData["Title"] = @StaticTextResource.ProvideFeedbackAndContact_ContactFindATender;
```
Comment (rmohammed-goaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:12]:
```suggestion
        ViewData["Title"] = StaticTextResource.ProvideFeedbackAndContact_ContactFindATender;
```

**Review by rmohammed-goaco — APPROVED** [review]
Approved with a couple of suggestions for consistency

**Review by andymantell — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 2 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml]: Done,

**Pull request merged**